src/helics/shared_api_library/MessageFederateExport.cpp

```diff
@@ -425,7 +425,7 @@ helics_message helicsEndpointGetMessage (helics_endpoint endpoint)
 
     if (endObj->lastMessage)
     {
-        helics_message mess{};
+        helics_message mess;
```

Comment:
I thought there was some reason this was an issue that the {} was required, but it may have been resolved in some of the HELICS 2.0 changes.    And I can't remember what it was right now.

Reply:
Would it be related to the default values for fields in the struct? The emptyMessage function above this one didn't use {}, but it seems to set all the fields -- would changing this to `helics_message mess = emptyMessage()` be something we would want to do?

Reply:
No, the intention was to not do anything with the fields that were just going to be filled in later.  There were times when the emptyMessage was needed, and other times when we just wanted space for the struct and the needed fields would be filled in later, so having a default initializer was redundent, and since the struct needed to work in a C context we couldn't make it behave like a class with the constructors and things.